cmd/ad_hoc_playbook.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+type AdHocPlaybook struct {
+	Playbook
+	files map[string]string
+}
+
+func (p *AdHocPlaybook) Run(playbookYml string, args []string) error {
+	// TODO: Panic if files is empty.
```

Comment:
Should this be done?

Reply:
done in https://github.com/roots/trellis-cli/pull/91/commits/6000a33e558d8825a04e2d27c07cdb48258da8a9